src/uu/runcon/src/runcon.rs

```diff
@@ -0,0 +1,453 @@
+// spell-checker:ignore (vars) RFILE
+
+use uucore::{executable, show_error, show_usage_error};
+
+use clap::{App, Arg};
+use selinux::{OpaqueSecurityContext, SecurityClass, SecurityContext};
+
+use std::borrow::Cow;
+use std::ffi::{CStr, CString, OsStr, OsString};
+use std::os::raw::c_char;
+use std::{io, ptr};
+
+mod errors;
+
+use errors::{report_full_error, Error, Result};
+
+static VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
Any reasons for preferring const instead of static?

Reply:
I was under the impression that it's generally preferred to use `const`. `const`s are inlined while `static`s are not, so there might be a performance improvement.

Reply:
As far as I know, `const`s are easier to optimize because they don't have a memory address, so their value can be shared or duplicated as needed by the optimizer. However,  `static`s have a stable memory address that needs to remain available, so duplicating them is more difficult to prove sound.

However, I used `static`s in order to remain consistent with the code of other commands in this project, which use `static`s most of the time. I agree with you, but @sylvestre might disagree.

Reply:
> which use statics most of the time.

I don't know if it's the real reason, but this might be because this project started before rust introduced `const`.

Reply:
So should I convert all `static`s to `const`s?

Reply:
Yes, I think that would be better.